### PR TITLE
Fix ios add icon. add generate splash image.

### PR DESCRIPTION
**Pull request by Nonsense331**

### CODE REVIEW

**Comment by boatrite** [comment]
:boat:

**Comment by boatrite** [comment]
Revoking the boat because you should add instructions to the README for the new splash task :)

**Comment by boatrite** [comment]
Then :shipit:

File: lib/high_five/thor/tasks/dist.rb [hunk 1]
```diff
@@ -55,7 +55,7 @@ def dist(platform)
             FileUtils.cp(@provisioning_profile, "#{ENV['HOME']}/Library/MobileDevice/Provisioning Profiles/#{uuid}.mobileprovision")
             system_or_die("rm -rf #{ios_path}/build/*")
             system_or_die(%Q(cd "#{ios_path}";
-              /usr/bin/xcodebuild -target "#{ios_target}" -configuration Release clean build "CONFIGURATION_BUILD_DIR=#{ios_path}/build" "CODE_SIGN_IDENTITY=#{@sign_identity}" PROVISIONING_PROFILE=$uuid))
+              /usr/bin/xcodebuild -target "#{ios_target}" -configuration Release build "CONFIGURATION_BUILD_DIR=#{ios_path}/build" "CODE_SIGN_IDENTITY=iPhone Distribution" SHARED_PRECOMPS_DIR=#{ios_path}/build/sharedpch ENABLE_BITCODE=NO "ARCHS=armv7 armv7s arm64" "VALID_ARCHS=armv7 armv7s arm64" SDKROOT=iphoneos9.2))
```
Comment (samsonasu) [inline lib/high_five/thor/tasks/dist.rb]: What's up with this?

File: lib/high_five/thor/tasks/android.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 require 'chunky_png'
 require 'high_five/android_helper'
 require 'nokogiri'
+require 'pry'
```
Comment (samsonasu) [inline lib/high_five/thor/tasks/android.rb]: remove this it probably won't work in some places

**Comment by samsonasu** [comment]
looks good except those two notes

**Comment by samsonasu** [comment]
fix and bump version to 0.3.14 please

**Pull request merged**